terraform/core/70-nec-housing.tf

```diff
@@ -0,0 +1,24 @@
+module "file_sync_destination_nec" {
+  source      = "git::https://github.com/LBHackney-IT/ce-file-sync-modules.git//destination?ref=64aa50415de498e360d2aa89694870ca810ea457" # v0.5.0
+  tags        = merge(module.tags.values, { "Environment" = lower(var.environment) })
```

Comment:
this lower is to make the tags on the buckets this module deploys compliant with the tagging strategy